src/ocsp.c

```diff
@@ -1074,34 +1074,39 @@ WOLFSSL_OCSP_CERTID* wolfSSL_d2i_OCSP_CERTID(WOLFSSL_OCSP_CERTID** cidOut,
 {
     WOLFSSL_OCSP_CERTID *cid = NULL;
 
-    if ((cidOut == NULL) || (derIn == NULL) || (length == 0))
-        goto err;
+    if ((cidOut != NULL) && (derIn != NULL) && (length > 0)) {
+
+        cid = *cidOut;
+
+        /* If a NULL is passed we allocate the memory for the caller. */
+        if (cid == NULL)
+            cid = (WOLFSSL_OCSP_CERTID*)XMALLOC(sizeof(*cid), NULL,
+                                                DYNAMIC_TYPE_OPENSSL);
+        else if (cid->rawCertId) {
+            XFREE(cid->rawCertId, NULL, DYNAMIC_TYPE_OPENSSL);
+            cid->rawCertId = NULL;
+            cid->rawCertIdSize = 0;
+        }
 
-    cid = *cidOut;
-    /* If a NULL is passed we allocate the memory for the caller. */
-    if (!cid) {
-        cid = (WOLFSSL_OCSP_CERTID*)XMALLOC(sizeof(*cid), NULL, DYNAMIC_TYPE_OPENSSL);
-        if (!cid) goto err;
-    }
-    else if (cid->rawCertId) {
-        XFREE(cid->rawCertId, NULL, DYNAMIC_TYPE_OPENSSL);
-        cid->rawCertId = NULL;
-        cid->rawCertIdSize = 0;
-    }
+        if (cid != NULL) {
+            cid->rawCertId = (byte*)XMALLOC(length, NULL, DYNAMIC_TYPE_OPENSSL);
+            if (cid->rawCertId != NULL) {
+                XMEMCPY (cid->rawCertId, *derIn, length);
```

Comment:
Removes spaces between function calls and the parameter list here and elsewhere. Not a coding style we use.